docs/index.rst

```diff
@@ -0,0 +1,9 @@
+Pyhound pdf
+===========
+
+
+This is the pdf harvester for pyhound.
+
+See the `[README]`_ for more information & examples.
+
+.. _[README]: https://github.com/Polyconseil/pyhound
```

Comment:
The current configuration of our Jenkins platform requires a documentation in every project.